LandLord-legacy/src/main/java/biz/princeps/landlord/OwnedLand.java

```diff
@@ -67,7 +66,7 @@ public String getOwnersString() {
         for (UUID uuid : itt) {
             names.add(Bukkit.getOfflinePlayer(uuid).getName());
         }
-        return itToString(names.iterator());
+        return String.join(", ", names);
```

Comment:
See comments in the equivalent class in `latest` (you could make the constant `protected` and add it to the super class or, probably even better, make it private in the super class and add a protected method `formatNames` that can be reused in both implementations then).